torch/_inductor/cpp_builder.py

```diff
@@ -1621,50 +1603,32 @@ def __init__(
-            compiler: str,
-            sources: str,
-            include_dirs_args: str,
-            definitions_args: str,
-            cflags_args: str,
-            ldflags_args: str,
-            libraries_args: str,
-            libraries_dirs_args: str,
-            passthrough_args: str,
-            output: str,
-        ) -> str:
-            if _IS_WINDOWS:
-                # https://learn.microsoft.com/en-us/cpp/build/walkthrough-compile-a-c-program-on-the-command-line?view=msvc-1704
-                # https://stackoverflow.com/a/31566153
-                cmd = (
-                    f"{compiler} {include_dirs_args} {definitions_args} {cflags_args} "
-                    f"{sources} {passthrough_args} {output}"
-                )
-                if self._do_link:
-                    cmd += f" /LD /link {libraries_dirs_args} {libraries_args} {ldflags_args}"
-                cmd = normalize_path_separator(cmd)
-            else:
-                cmd = (
-                    f"{compiler} {sources} {definitions_args} {cflags_args} "
-                    f"{include_dirs_args} {passthrough_args} {output}"
+        if _IS_WINDOWS:
+            # https://learn.microsoft.com/en-us/cpp/build/walkthrough-compile-a-c-program-on-the-command-line?view=msvc-1704
+            # https://stackoverflow.com/a/31566153
+            cmd = (
+                f"{self._compiler} {self._include_dirs_args} {self._definitions_args} "
+                f"{self._cflags_args} {self._sources_args} "
+                f"{self._passthrough_parameters_args} {self._output}"
+            )
+            if self._do_link:
+                cmd += (
+                    f" /LD /link {self._libraries_dirs_args} {self._libraries_args} "
+                    f"{self._ldflags_args}"
                 )
-                if self._do_link:
-                    cmd += f" {ldflags_args} {libraries_args} {libraries_dirs_args}"
-            return cmd
-
-        command_line = format_build_command(
-            compiler=self._compiler,
-            sources=self._sources_args,
-            include_dirs_args=self._include_dirs_args,
-            definitions_args=self._definitions_args,
-            cflags_args=self._cflags_args,
-            ldflags_args=self._ldflags_args,
-            libraries_args=self._libraries_args,
-            libraries_dirs_args=self._libraries_dirs_args,
-            passthrough_args=self._passthrough_parameters_args,
-            output=self._output,
+            return normalize_path_separator(cmd)
+
+        cmd = (
+            f"{self._compiler} {self._sources_args} {self._definitions_args} "
+            f"{self._cflags_args} {self._include_dirs_args} "
+            f"{self._passthrough_parameters_args} {self._output}"
         )
-        return command_line
+        if self._do_link:
+            cmd += (
+                f" {self._ldflags_args} {self._libraries_args} "
+                f"{self._libraries_dirs_args}"
+            )
+        return cmd
```

Comment:
This is functionally neutral; it just removes an interior function that added extra indentation and was only called here.